ansible/group_vars/all.yml

```diff
@@ -2,13 +2,16 @@
 edu_sharing_host: 192.168.98.101
 edu_sharing_domain: '{{edu_sharing_host}}'
 edu_sharing_url: '{{ edu_sharing_protocol | default("http") }}://{{edu_sharing_host}}/edu-sharing'
-edu_version: 6.0.1
+edu_version: 8.0.0
```

Comment:
This doesn't / shouldn't belong in this PR, but for my sake we can update it

Reply:
Hello @mirjan-hoffmann 

Yes, you are right, even that we already have in prod 8.0.0, we can update it but this is not part of pull request and I will remove